aerospike/tests/conftest.py

```diff
@@ -55,6 +55,7 @@ def dd_environment():
     with docker_run(
         COMPOSE_FILE,
         conditions=[CheckDockerLogs(COMPOSE_FILE, ['service ready: soon there will be cake!']), WaitFor(init_db)],
+        attempts=2,
```

Comment:
why 2? (and not 3..or 4...?)

Reply:
Right now we have only 1 attempt and the tests do not fail that often. I think 2 should be OK and if it fail again with 2 we might need to update something in the test itself, and not just retry

Reply:
The exact issue we're experiencing is trying to pull some docker image by timeout or starting some container inside docker-compose?

Reply:
The problem here is during the `docker pull` command, we get an unexpected error

Reply:
In this case we are probably experiencing errors beyond our control (network, connection cuts, timeouts...) and that can be solved in a second attempt due to the cached data (image layers) of the first attempt. The optimal solution in these cases would be to use a local docker registry to minimize these types of errors, but it is a more complex solution that requires time and infrastructure.